src/test/scala/uk/gov/hmrc/test/ui/pages/section2/AuthorisationsListPage.scala

```diff
@@ -37,4 +37,6 @@ object AuthorisationsListPage extends BasePage {
   // Yes => fillPage(yes)
 
   override def fillPage(values: String*): Unit = selectYesOrNoRadio(values(yesNo))
+
+  def removeAuthCode(index: Int): Unit = clickByCssSelector(s"#holder-table-row$index-remove_button>a")
```

Comment:
No need to change this. I just want to point out an alternative implementation...
```
def removeAuthCode(index: Int): Unit =
  findChildByTag(findElementById("holder-table-row$index-remove_button"), "a")
```